app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -109,6 +120,50 @@ def parse_build_pod(build_pod)
                                                            build_pod.status.config.try(:name))
         new_result
       end
+
+      def parse_exposed_ports(exposed_ports)
+        exposed_ports.to_h.keys.each_with_object({}) do |port, h|
+          n, p = port.to_s.split('/', 2)
+          h[n] = p
+        end
```

Comment:
@enoodle I don't understand this. You create an hash with `protocol => port`? E.g. `{'tcp' => 8080}`? What if there are multiple ports of the same protocol are exposed? Shouldn't this just be a list of exposed ports? `['tcp/8080', 'tcp/80', 'udp/1111' ...]`